integration-tests/jpa-mysql/pom.xml

```diff
@@ -196,13 +196,11 @@
                                         <tmpfs>/var/lib/mysql</tmpfs>
                                         <wait>
                                             <!-- good docs found at: http://dmp.fabric8.io/#build-healthcheck -->
```

Comment:
This line seems no longer relevant,  `<postStart>mysqladmin ...` is not mentioned there

Reply:
Good catch!, I'll update it